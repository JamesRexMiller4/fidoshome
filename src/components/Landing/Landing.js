import React from 'react';
import './Landing.scss';
import Header from '../Header/Header';

const Landing = () => {
  return ( 
    <main>
      <Header />
      <section>
        <h2 id='welcome-1' className='landing-welcome-h2'>Give your new best friend</h2>
        <h2 id='welcome-2' className='landing-welcome-h2'>the home they always dreamed of</h2>
      </section>
    </main>
  );
}

export default Landing;