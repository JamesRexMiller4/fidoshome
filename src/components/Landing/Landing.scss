.landing-welcome-h2 {
  font-family: 'Libre Baskerville', serif;
  font-weight: bold;
  font-size: 40px;
  color: rgb(240, 238, 238,);
}

.welcome-section {
  height: 60vh;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  align-items: center;
}

#welcome-1 {
  animation: fadeIn 2.5s linear;
}

#welcome-2 {
  opacity: 0;
  animation-name: fadeIn;
  animation-duration: 2.5s;
  animation-delay: 2s;
  animation-fill-mode: forwards;
}

.btn {
  opacity: 0;
  width: 15vw;
  font-family: 'Libre Baskerville', serif;
  font-weight: bold;
  font-size: 40px;
  color: rgb(205, 174, 2);
  border: 5px solid;
  border-color: rgb(205, 174, 2);
  border-radius: 30px;
  cursor: pointer;
  animation-name: pulse;
  animation-delay: 4s;
  animation-duration: 2s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;

  &:hover {
    opacity: 1;
    animation: none;
    font-size: 45px;
    color: rgb(242, 212, 46);
    border-color: rgb(242, 212, 46);
  }
}

@keyframes fadeIn {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}

@keyframes pulse {
  from {
    opacity: .2
  }

  to {
    opacity: 1
  }
}