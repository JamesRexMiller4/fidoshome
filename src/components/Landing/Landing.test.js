import React from 'react';
import { shallow} from 'enzyme';
import Landing from './Landing.js';

describe('Landing Component', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Landing />)
    expect(wrapper).toMatchSnapshot();
  });
})