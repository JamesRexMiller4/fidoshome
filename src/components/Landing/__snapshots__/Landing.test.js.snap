// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Component should match the snapshot 1`] = `
<main>
  <header
    className="header-landing"
  >
    <h1
      className="header-h1"
    >
      Fido's
    </h1>
    <img
      alt="dog"
      className="header-profile"
      src="regalpuppy.jpg"
    />
    <h1
      className="header-h1"
    >
      Home
    </h1>
  </header>
  <section
    className="welcome-section"
  >
    <h2
      className="landing-welcome-h2"
      id="welcome-1"
    >
      Give your new best friend
    </h2>
    <Link
      to="/home"
    >
      <button
        className="btn"
        id="enter-btn"
        type="button"
      >
        Enter
      </button>
    </Link>
    <h2
      className="landing-welcome-h2"
      id="welcome-2"
    >
      the home they always dreamed of
    </h2>
  </section>
</main>
`;
